Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionScreenSpaceComposite.azsl

```diff
@@ -35,23 +42,20 @@ ShaderResourceGroup PassSrg : SRG_PerPass
         AddressV = Clamp;
         AddressW = Clamp;
     };
-
-    // the max roughness mip level for sampling the previous frame image
-    uint m_maxMipLevel;
 }
 
 #include <Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli>
 
-float3 SampleReflection(float2 reflectionUV, float mip, float depth, float3 normal, uint2 invDimensions)
+float3 SampleReflection(float2 reflectionUV, float depth, float3 normal, uint2 invDimensions)
 {  
     const float DepthTolerance = 0.001f;
```

Comment:
minor-> const int mip = 0; and then use that variable like before

Reply:
This function is only using SampleLevel with mip 0 now, as we do in many other shaders, so introducing a variable for the mip would make it harder to follow.